Invalid version string reported when using the kcp executable

I'm performing the following steps:

1. Clone the repo
2. git checkout tag/v0.5.0-alpha.1
3. cd kcp
4. make build
5. add newly built executable to PATH
6. kcp --help

Step 6 outputs this:
```
panic: version string "+kcp-v0.5.0-alpha.1" doesn't match expected regular expression: "^v(\d+\.\d+\.\d+)"
```

I am using go version 1.17 on MacOS.

--- a/Makefile
+++ b/Makefile
@@ -71,8 +71,15 @@ LDFLAGS := \
 all: build
 .PHONY: all
 
+pre-build-checks:
+ifeq ($(and $(KUBE_MAJOR_VERSION),$(KUBE_MINOR_VERSION)),)
+	$(info Kubernetes version not set. Ensure jq is installed.)
+	exit 1
+endif
+.PHONY: pre-build-checks
+
 build: WHAT ?= ./cmd/...
-build: ## Build the project
+build: pre-build-checks ## Build the project
 	go build -ldflags="$(LDFLAGS)" -o bin $(WHAT)
 .PHONY: build
 

